index.php

```diff
@@ -34,6 +34,47 @@
         <?= $winesToDisplay; ?>
     </div>
 
+    <form method="post" action="validate.php">
+        <p>Wine Name:</p><br>
```

Comment:
Change back to label tag!  
For convention sake... in your labels for mandatory fields, add *.  E.g. Wine Name *: 

I will not litter your page with the same request!